.github/workflows/fdroid.yml

```diff
@@ -122,6 +122,11 @@ jobs:
       - name: Update repo
         env:
           GH_TOKEN: ${{ steps.retrieve-secrets.outputs.github-pat-bitwarden-devops-bot-repo-scope }}
-        if: ${{ (github.event_name == 'schedule' || inputs.dry-run == 'false') && steps.run-metascoop.outputs.has_changes == 'true' }}

```

Comment:
Looking at this now, this was probably comparing bools with strings. The new changes will work due to converting them to strings first. If that's correct, this should work too:

`if: ${{ (github.event_name == 'schedule' || inputs.dry-run == false) && steps.run-metascoop.outputs.has_changes == true }}`

Reply:
Good catch! I actually prefer having the verbose output though. It gives us a better idea of why changes are or are not being saved.